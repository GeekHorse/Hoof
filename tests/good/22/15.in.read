hello 
new 
ok 
new 
ok 
new 
ok 
ok 
ok 
ok   baz zab 
ok 
