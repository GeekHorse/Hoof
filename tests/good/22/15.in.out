new right foo oof done
new down bar rab done
new down baz zab done
